### PR TITLE
docs: adds terms to wordlist

### PR DESCRIPTION
**Pull request by chri2547**
This PR adds "KPI" and "KPIs" to the wordlist so that they won't be flagged by the linter.

### CODE REVIEW

**Review by jdbaldry — COMMENTED** [review]
Thank you for the contribution! Is this abbreviation also worth excluding from the https://github.com/grafana/writers-toolkit/blob/main/vale/Grafana/Acronyms.yml rule?

My leaning is that we should probably still explain this term on its first use in each page.

File: vale/dictionaries/en_US-grafana.dic [hunk 1]
```diff
@@ -36,6 +36,8 @@ JMeter/ po:noun
 Jsonnet/ po:noun
 JUnit/ po:noun
 Kinesis/ po:noun
+KPI/ po:noun
+KPIs/ po:noun
```
Comment (jdbaldry) [inline vale/dictionaries/en_US-grafana.dic:40]: This is just a nit of how the dictionary works but with a `/S` we can have a single entry for this work plus the pluralization which comes from the affix rule in https://github.com/grafana/writers-toolkit/blob/main/vale/dictionaries/en_US-grafana.aff#L20-L24.

```suggestion
KPI/S po:noun
```

Comment (chri2547) [reply vale/dictionaries/en_US-grafana.dic]: @jdbaldry oh cool, thanks!

Comment (jdbaldry) [reply vale/dictionaries/en_US-grafana.dic]: I've already applied the suggestion but I've left the comment unresolved as explanation.

**Pull request merged**